packages/webpack/src/utils/postcss.js

```diff
@@ -6,6 +6,7 @@ import cloneDeep from 'lodash/cloneDeep'
 import createResolver from 'postcss-import-resolver'
 
 import { isPureObject } from '@nuxt/utils'
+import { requireModule } from '@nuxt/cli/src/utils'
```

Comment:
By design, No non-distro package should depend on `@nuxt/cli`

Reply:
Whoops. That was not intended 🙈